modules/gdscript/gdscript_parser.cpp

```diff
@@ -269,6 +269,9 @@ void GDScriptParser::override_completion_context(const Node *p_for_node, Complet
 	context.current_argument = p_argument;
 	context.node = p_node;
 	context.parser = this;
+	if (!completion_call_stack.is_empty()) {
+		context.call = completion_call_stack.get(completion_call_stack.size() - 1);
```

Comment:
```suggestion
		context.call = completion_call_stack.back()->get();
```
Avoid random access here, that is is `O(n)`, this is `O(1)`